### DISCUSSION TITLE
Campaigns csv download using API

### DISCUSSION THREAD

**Discussion by CoolDev0723** [amzn/ads-advanced-tools-docs#200] [Q&A]
Advertising->Campaign Manager->Portfolios
From the url below, I can download the list of campaigns for each portfolio in CSV format.
https://advertising-japan.amazon.com/cm/portfolios?entityId=ENTITY2FLVA6EUQ5BZ1
https://advertising-japan.amazon.com/cm/portfolios/A2XZTPW4RPRFF1

Can I download this csv using API?
I checked the API documentation, but couldn't find a matching endpoint.
I would appreciate it if anyone could teach me.
Thanks in advance.

![image](https://github.com/amzn/ads-advanced-tools-docs/assets/54260086/34b514a7-d2aa-40a7-9018-cf87bd83e4c3)

![image](https://github.com/amzn/ads-advanced-tools-docs/assets/54260086/e118433a-f0e7-4f4c-9085-3fb855518d25)

**Comment by drewxa** [comment]
Consider using the exports feature in the Amazon Advertising API, as detailed in their documentation here: https://advertising.amazon.com/API/docs/en-us/guides/exports/get-started. This feature facilitates the retrieval of all campaigns within a profile. The data is initially in JSON format, but converting it to CSV is a straightforward process.

**Reply by CoolDev0723** [reply]
Thank you for your cooperation.

{{api_url}}/campaigns/export
{{api_url}}/adGroups/export

I checked the endpoints above, but their output is not what I want.

Menu->Advertising->Campaign Manager->Portfolios->Portfolio
I would like to obtain the same information as the csv file output from the above page through API.
Please check again. Thanks in advance.

**Reply by drewxa** [reply]
Could you provide more specific details about the data that is absent in the campaign exports but is present in the specified CSV file? A more in-depth understanding of the missing information may enable me to offer assistance or insights that could be helpful in resolving the issue.
